clients/manifest.go

```diff
@@ -364,6 +378,18 @@ func CreateClippedPlaylist(origManifest m3u8.MediaPlaylist, segs []*m3u8.MediaSe
 		if i != 0 && i != (len(segs)-1) {
 			s.URI = "../" + s.URI
 		}
+		// Remove PROGRAM-DATE-TIME tag from first clipped segment so that player doesn't
+		// run into seek issues or display incorrect times on playhead
+		if i == 0 {
+			s.ProgramDateTime = t
+		}
+		// Add a DISCONTINUITY tag to let hls players know about different encoding between
+		// segments. But don't do this if there's a single segment in the clipped manifest
+		// or contains total two segments in the clipped manifest (i.e. back to back segments
+		// were run through ffmpeg)
+		if i-1 == 0 && totalSegs > 2 {
```

Comment:
Do we also need this before the final segment?

Reply:
No, because the final segment is not re-encoded to shares the same params as the segment before it -- so a/v decoders don't need to be re-initialized.